packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -2,16 +2,16 @@
 
 const {
   providers: { createLocalFileDestinationProvider },
-} = require('@strapi/data-transfer/lib/file');
+} = require('@strapi/data-transfer').file;
```

Comment:
Were these left like this on purpose, or should we keep all of these together as well?

```
const {
  file: {
    providers: { createLocalFileDestinationProvider },
  },
  strapi: {
    providers: { createLocalStrapiSourceProvider },
  },
  engine: {
    createTransferEngine,
    errors: { TransferEngineTransferError },
  },
} = require('@strapi/data-transfer');
```

Reply:
I left them as you split them but I would rather do it like you show instead personnaly :)